src/components/GetReport.jsx

```diff
@@ -1,76 +1,145 @@
 import React from "react";
 import { Formik, Form, Field } from "formik";
-import { GetResponseErrors } from "../utilities/CitysourcedResponseHelpers";
 import FormContainer from './FormContainer';
+import { SetFieldValues } from "../utilities/FormHelpers";
 import { GetReportByID } from '../services/authService';
+import '../css/inside-responsive.min.css';
+import '../css/citysourced-reporter.min.css';
+import SeButton from "./SeButton";
+import Note from './Note';
 import { Go, Routes } from "../Routing";
 
-const GetReport = props => {
-	const userGetReport = async (values) => {
+const GetReport = (props) => {
+    
+	const handleChange = changeEvent => {
+		SetFieldValues(props, {trackingNumber: changeEvent.target.value});
+	};
 
+	const userGetReport = async (trackingNumber) => {
+		const response = await GetReportByID(trackingNumber);
+		Go(props, Routes.ReportStatus, response)
+		return response;
+	}
 
-		try {
-			const response = await GetReportByID(values.ReportID);
-			if (response.data.ErrorsCount > 0) {
-				const errorsReturned = GetResponseErrors(response);
-				props.Field.ErrorMsg = errorsReturned;
-			}
-			else {
-				Go(props, Routes.AdditionalInformation);
-			}
+	const checkTrackingNumber = () =>{
+		const { trackingNumber } = props.values;
+	
+		if(RegExp(/^(ACCMP)/i).test(trackingNumber)){
+			buildNote('?&Module=Enforce');
+		}
+		else if (RegExp(/^(CC|CRH|CS|PP|TS|CE|CP|CB|CG)\d+$/i).test(trackingNumber)){
+			buildNote('?&Module=Enforcement');
+		}	
+		else if (RegExp( /^\d+$/i).test(trackingNumber)){
+			userGetReport(trackingNumber);	
 		}
-		catch (ex) {
-			if (ex.response && ex.response.status === 400) {
-				props.Form.Field.email.errors = ex.response.data
-			}
+		else
+		{
+			buildNote('BadID');
```

Comment:
not blocking for now. but since you use `BadID` in a couple different places, it should be stored in a variable or config somewhere. Routes might make sense in this case

Reply:
I made these just a const since its used only on this page and only twice. Not sure it needs to go beyond this page into a config since its tightly coupled with this page logic.

Reply:
that makes sense to me.